### PR TITLE
nu am pus assertul ca nu mai mere site-ul

### PR DESCRIPTION
**Pull request by CatalinVDN**

### CODE REVIEW

File: pages/flightsPage.py [hunk 1]
```diff
@@ -1,9 +1,51 @@
+from ast import Try
```
Comment (adrianmaciuc) [inline pages/flightsPage.py:1]: please remove

File: pages/flightsPage.py [hunk 2]
```diff
@@ -1,9 +1,51 @@
...
 from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.common.exceptions import NoSuchElementException
```
Comment (adrianmaciuc) [inline pages/flightsPage.py:5]: What are your intentions with this?

File: pages/flightsPage.py [hunk 3]
```diff
@@ -1,9 +1,51 @@
...
+    TICKETQTY = (By.CSS_SELECTOR, ".dropdown-item")
+    QTYINC = (By.CSS_SELECTOR, ".qtyInc")
+    NORESULT = (By.CSS_SELECTOR, "#fadein > div.container.text-center")
+
+
+    def load(self):
+        self.browser.get(FlightsPage.URL)
+        self.browser.maximize_window()
+
+    def insert_departure_arrival(self, locator_value, City):
```
Comment (adrianmaciuc) [inline pages/flightsPage.py:30]: best practice to write lowercase

**Review by adrianmaciuc — APPROVED** [review]
Very NICE !

File: pages/flightsPage.py [hunk 4]
```diff
@@ -1,9 +1,51 @@
...
 class FlightsPage(BasePage):
-    URL = "https://www.phptravels.net/flights"
\ No newline at end of file
+    URL = "https://www.phptravels.net/flights"
+    FLYFROM = (By.NAME, "from")
+    FLYTO = (By.NAME, "to")
+    DATE = (By.NAME, "depart")
+    PASSENGERS = (By.CSS_SELECTOR, ".dropdown-contain")
+    SEARCH = (By.CSS_SELECTOR, ".ladda-label")
+    DROPDOWN = (By.CSS_SELECTOR, ".autocomplete-results.intro.troll")
```
Comment (AnubisMRM) [inline pages/flightsPage.py:19]: DROPDOWN si DROPDOWN2 ar putea fi ceva mai sugestive, ca sa nu fie nevoie sa deschidem pagina si sa cautam acele elemente. Ma gandeam la ceva de genul FLYING_FROM_DROPDOWN / TO_DESTINATION_DROPDOWN.

**Review by AnubisMRM — COMMENTED** [review]
When running test_flights.py locally, the test fails at line 23 (self.flights.wait_for_element(self.flights.TICKETQTY). I'm not a for loop expert, so I am not sure how to fix it.

**Comment by adrianmaciuc** [comment]
> When running test_flights.py locally, the test fails at line 23 (self.flights.wait_for_element(self.flights.TICKETQTY). I'm not a for loop expert, so I am not sure how to fix it.

Can confirm error. Did debug and found the same.

Exception has occurred: TimeoutException

File "[C:\python_projects\code_along\pages\flightsPage.py]()", line 63, in <module>
    flights.wait_for_element(FlightsPage.TICKETQTY)

File: pages/flightsPage.py [hunk 5]
```diff
@@ -16,9 +14,9 @@ class FlightsPage(BasePage):
     DATE = (By.NAME, "depart")
     PASSENGERS = (By.CSS_SELECTOR, ".dropdown-contain")
     SEARCH = (By.CSS_SELECTOR, ".ladda-label")
-    DROPDOWN = (By.CSS_SELECTOR, ".autocomplete-results.intro.troll")
-    DROPDOWN2 = (By.CSS_SELECTOR, "._1.autocomplete-wrapper.row_2 > .autocomplete-results.intro.troll")
-    TICKETQTY = (By.CSS_SELECTOR, ".dropdown-item")
+    DROPDOWNDEPARTURES = (By.CSS_SELECTOR, ".autocomplete-results.intro.troll")
+    DROPDOWNARRIVAL = (By.CSS_SELECTOR, "._1.autocomplete-wrapper.row_2 > .autocomplete-results.intro.troll")
+    TICKETQTY = (By.CSS_SELECTOR, "#onereturn > div.col-lg-1.pr-0 > div > div > div > div > div.dropdown-item")
```
Comment (adrianmaciuc) [inline pages/flightsPage.py:19]: pffff o mizerie

**Review by adrianmaciuc — APPROVED** [review]

File: pages/flightsPage.py (hunk 5 above)
Comment (CatalinVDN) [reply pages/flightsPage.py]: :(

**Review by AnubisMRM — APPROVED** [review]

**Pull request merged**